databaseAnnotator/src/main/resources/application.properties.EXAMPLE

```diff
@@ -5,6 +5,7 @@ databaseannotator.pass=
 databaseannotator.driver=com.mysql.jdbc.Driver
 databaseannotator.connection_string=jdbc:mysql://<HOST>/<DATABASE>
 
-genomenexus.hgvs=http://<HOST>:<PORT>/variant_annotation/hgvs/
+genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
+genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=cancerHotspots
+genomenexus.enrichment_fields=hotspots,mutation_assessor
```

Comment:
This change was present in #42 .. and must be coordinated with the change to GenomeNexusImpl.java in order to work correctly, so I think this should be removed from here. [or GenomeNexusImpl changes should be added here and #42 can be dropped]